WalletWasabi.Tests/UnitTests/Clients/WasabiClientTests.cs

```diff
@@ -0,0 +1,94 @@
+using NBitcoin;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Missing using statement
```suggestion
using System.Threading.Tasks;
using WalletWasabi.Tests.UnitTests.Filters;
```